pypeit/calibrations.py

```diff
@@ -560,47 +562,47 @@ def get_flats(self):
         # 3) there is no master or no user supplied flat, generate the flat
         if self.mspixelflat is None and len(pixflat_image_files) != 0:
             # Run
-            self.mspixelflat, self.msillumflat = self.flatField.run(show=self.show,
-                                                                    maskslits=self.tslits_dict['maskslits'])
+            self.mspixelflat, self.msillumflat = self.flatField.run(show=self.show, debug=True)
 
-            # If we tweaked the slits, update the tilts_dict and
-            # tslits_dict to reflect new slit edges
-            if self.par['flatfield']['tweak_slits']:
-                msgs.info('Using slit boundary tweaks from IllumFlat and updated tilts image')
-                self.tslits_dict = self.flatField.tslits_dict
-                self.tilts_dict = self.flatField.tilts_dict
+            # Objects should point to the same data
```

Comment:
Don't forget to remove this. I'm not following exactly as I didn't think assert worked with objects in the same way that copy does not work or is problematic.

Reply:
https://dbader.org/blog/difference-between-is-and-equals-in-python

This is just testing that, e.g., `self.slits` and `self.flatField.slits` are the same object.  I.e., we don't need to update one based on the other because they're both pointing to the same object and have the same data by definition.